wagtail/admin/tests/pages/test_copy_page.py

```diff
@@ -555,3 +555,35 @@ def test_page_copy_alias_post_copy_subpages(self):
 
         # treebeard should report no consistency problems with the tree
         self.assertFalse(any(Page.find_problems()), 'treebeard found consistency problems')
+
+    def test_page_copy_alias_post_without_destination_publish_permission(self):
```

Comment:
Name is the wrong way round:
```suggestion
    def test_page_copy_alias_post_without_source_publish_permission(self):
```